master.css

```diff
@@ -350,7 +365,18 @@ label[for="sort"] {
     width: 260px;
 }
 .num-features {
-    position: absolute; top: -15px; right: 0px; font-size: 11px;
+    position: absolute;
+    top: -15px;
+    right: 0px;
+    font-size: 11px;
+    left: 0px;
+    background-color: #eee;
```

Comment:
I'm not sure why this is necessary, except that it puts somewhat ungainly gray boxes around the feature% totals in every table *except* ES6.